Makefile

```diff
@@ -64,8 +64,9 @@ gitlint:
 tools:
 	go get github.com/rakyll/statik
 
-update_registry:
+update_charts:
```

Comment:
I think we could keep `update_registry` and add `update_mlflow` and have `update_charts` calling both. In that way, if I only need to update one of them I do not need to do it for both.